azext_edge/edge/providers/check/base/pod.py

```diff
@@ -81,11 +82,7 @@ def _decorate_pod_condition(condition: bool) -> Tuple[str, str]:
 
         pod_conditions = [
             f"{target_service_pod}.status.phase",
-            f"{target_service_pod}.status.conditions.ready",
-            f"{target_service_pod}.status.conditions.initialized",
-            f"{target_service_pod}.status.conditions.containersready",
-            f"{target_service_pod}.status.conditions.podscheduled",
-            f"{target_service_pod}.status.conditions.podreadytostartcontainers",
+            f"{target_service_pod}.status.conditions",
```

Comment:
personally would like to have an even simpler one line for the entire pod state (is it good? is it bad?) in summary view

But if we want to show the phase vs condition - this is ok

Reply:
updated~